doc/sphinxext/custom_roles.py

```diff
@@ -1,16 +1,24 @@
 from docutils import nodes
 from os.path import sep
+from matplotlib import rcParamsDefault
 
 
 def rcparam_role(name, rawtext, text, lineno, inliner, options={}, content=[]):
-    rendered = nodes.Text('rcParams["{}"]'.format(text))
+
+    param = rcParamsDefault.get(text)
+    if isinstance(param, str):
+        txt = f' : "{param}"'
```

Comment:
1) no space before the colon? (or use "=")
2) you can probably get away with `{param!r}` (aka `{repr(param)}`) which will give you the quotes for strings.